### PR TITLE
Feature/edit from filelist

### PR DESCRIPTION
**Pull request by klaseskilson**
This allows files to be edited and deleted from the file list, without reloading the whole page.

skarmavbild 2015-05-22 kl 15 38 17

### CODE REVIEW

File: app/controllers/files/files_controller.rb [hunk 1]
```diff
@@ -63,7 +65,7 @@ def index
   #   :error:string if @error =>
   #
   def update
-    @file = BruseFile.find(params[:bruse_file][:id])
+    @file = current_user.bruse_files.find(params[:bruse_file][:id])
```
Comment (danielronnkvist) [inline app/controllers/files/files_controller.rb:68]: nice! :zap:

**Comment by danielronnkvist** [comment]
Cool! Ready for merge :pineapple:

File: app/controllers/files/files_controller.rb (hunk 1 above)
Comment (klaseskilson) [inline app/controllers/files/files_controller.rb:68]: :ok_hand:

**Comment by ronjagrosz** [comment]
intebra
:cry:

**Comment by klaseskilson** [comment]
Hmm. Not too good looking, no. 🙈 I'll take a look at it.

**Comment by klaseskilson** [comment]
The issues regarding row-breaking icons in the tools area is that tables are hard to control. I'm not sure how we should handle this, but one idea could be to move everything from the `table`-based structure we have today to a `div`-based structure. I think that could be done in an other PR, but it could help us get rid of this unreliable table we have today.

**Comment by ronjagrosz** [comment]
looks good!

**Comment by danielronnkvist** [comment]
Looks good, will merge! Regarding migrating to a `div` based structure I think that has a low priority until Friday.

**Pull request merged**